build/Targets/Settings.props

```diff
@@ -124,6 +124,7 @@
     <When Condition="'$(MSBuildProjectExtension)' == '.vbproj' OR '$(Language)' == 'VB' OR '$(ProjectLanguage)' == 'VB'">
       <PropertyGroup>
         <ProjectLanguage>VB</ProjectLanguage>
+        <LangVersion>Latest</LangVersion>
```

Comment:
From discussion with Jason, I now agree with a specific downside of LangVersion=latest. When we introduce 7.2 language version in the master branch, the bootstrap compiler would allow 7.2 features to be used in the IDE code, as they are developed.
We could discuss whether that's a good thing, but, in any case, that is not what I intended with this change. I'll set LangVersion=7.1.